internal/models/factor.go

```diff
@@ -121,6 +128,9 @@ type Factor struct {
 	Secret       string      `json:"-" db:"secret"`
 	FactorType   string      `json:"factor_type" db:"factor_type"`
 	Challenge    []Challenge `json:"-" has_many:"challenges"`
+	// TODO: Encrypt this. Maybe store as string
```

Comment:
Not necessary to encrypt this, it's public data.

Reply:
Makes sense, stale comment. Let's remove this yeah